easybuild/easyconfigs/v/vConTACT2/vConTACT2-0.11.3-foss-2022a.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'vConTACT2'
+local_commit = 'c0413a6c92e86c58d624b4eb2e32687ceca89bd6'
+version = '0.11.3'
+
+homepage = 'https://bitbucket.org/MAVERICLab/vcontact2'
+description = """vConTACT2 is a tool to perform guilt-by-contig-association classification of
+ viral genomic sequence data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('Biopython', '1.79'),
+    ('networkx', '2.8.4'),
+    ('scikit-learn', '1.1.2'),
+    ('PyTables', '3.8.0'),
+    ('GitPython', '3.1.27'),  # required by clusterone
+]
+
+use_pip = True
+
+exts_list = [
+    ('click-log', '0.4.0', {
+        'checksums': ['3970f8570ac54491237bcdb3d8ab5e3eef6c057df29f8c3d1151a51a9c23b975'],
+    }),
+    ('coreschema', '0.0.4', {
+        'checksums': ['9503506007d482ab0867ba14724b93c18a33b22b6d19fb419ef2d239dd4a1607'],
+    }),
+    ('itypes', '1.2.0', {
+        'checksums': ['af886f129dea4a2a1e3d36595a2d139589e4dd287f5cab0b40e799ee81570ff1'],
+    }),
+    ('uritemplate', '4.1.1', {
+        'checksums': ['4346edfc5c3b79f694bccd6d6099a322bbeb628dbf2cd86eea55a456ce5124f0'],
+    }),
+    ('coreapi', '2.3.3', {
+        'checksums': ['46145fcc1f7017c076a2ef684969b641d18a2991051fddec9458ad3f78ffc1cb'],
+    }),
+    ('coreapi-cli', '1.0.9', {
+        'checksums': ['d76b5320ea9ad04d922001ac8897b903a7a359c08a53822f819134c50aa22c5f'],
+    }),
+    ('iso8601', '1.1.0', {
+        'checksums': ['32811e7b81deee2063ea6d2e94f8819a86d1f3811e49d23623a41fa832bef03f'],
+    }),
+    ('raven', '6.10.0', {
+        'checksums': ['3fa6de6efa2493a7c827472e984ce9b020797d0da16f1db67197bcc23c8fae54'],
+    }),
+    ('terminaltables', '3.1.10', {
+        'checksums': ['ba6eca5cb5ba02bba4c9f4f985af80c54ec3dccf94cfcd190154386255e47543'],
+    }),
+    ('tzdata', '2022.7', {
+        'checksums': ['fe5f866eddd8b96e9fcba978f8e503c909b19ea7efda11e52e39494bad3a7bfa'],
+    }),
+    ('pytz-deprecation-shim', '0.1.0.post0', {
+        'sources': ['pytz_deprecation_shim-%(version)s.tar.gz'],
+        'checksums': ['af097bae1b616dde5c5744441e2ddc69e74dfdcb0c263129610d85b87445a59d'],
+    }),
+    ('tzlocal', '4.2', {
+        'checksums': ['ee5842fa3a795f023514ac2d801c4a81d1743bbe642e3940143326b3a00addd7'],
+    }),
+    ('clusterone', '0.15.3', {
+        'checksums': ['ce253e778c68aa1c8f408ac2b835caca62d90b9c9206bf9f56819ee97b198cf9'],
+        # remove strict version requirements for required Python packages,
+        # remove requirement for enum34 (not needed when using Python >= 3.4)
+        'preinstallopts': "sed -i 's/==/>=/g' setup.py && sed -i '/enum34/d' setup.py && ",
```

Comment:
@boegel I was just checking if it's possible to add some sanity check commands, and then ran into this:
```
$ just -h
Traceback (most recent call last):
  File "/data/apps/software/vConTACT2/0.11.3-foss-2022a/bin/just", line 5, in <module>
    from clusterone.clusterone_cli import main
  File "/data/apps/software/vConTACT2/0.11.3-foss-2022a/lib/python3.10/site-packages/clusterone/clusterone_cli.py", line 81, in <module>
    @click_log.init(__name__)
AttributeError: module 'click_log' has no attribute 'init'
```
So the requirements may have to be somewhat stricter anyway, I guess...

Reply:
Nice catch!

Fixed in 3ff5fcdda7